src/actions/receive-profile.js

```diff
@@ -1460,7 +1460,14 @@ export function getProfilesFromRawUrl(
 ): ThunkAction<Promise<Profile | null>> {
   return async (dispatch, getState) => {
     const pathParts = location.pathname.split('/').filter(d => d);
-    let dataSource = ensureIsValidDataSource(pathParts[0]);
+    let possibleDataSource = pathParts[0];
+
+    // Treat from-addon as from-browser.
+    if (possibleDataSource === 'from-addon') {
+      possibleDataSource = 'from-browser';
+    }
```

Comment:
Can you please add a comment with the Firefox version when this will be changed? So that it's easier to know when we'll be able to remove this in the future (when the last ESR with `/from-addon` stops being supported possibly?). Presumably v92 (we can put it at v93 in case this lands after the merge date).

Reply:
Good point. Let's plan to put it in Firefox 93; this should give us enough time to deploy this.